app/webpack/observations/identify/components/observation_modal.jsx

```diff
@@ -36,6 +36,11 @@ class ObservationModal extends React.Component {
     };
   }
 
+  componentDidMount( ) {
+    const domNode = ReactDOM.findDOMNode( this );
+    $( ".IdentificationForm textarea,.CommentForm textarea", domNode ).textcompleteUsers( );
```

Comment:
This isn't working for me, I suspect b/c the textareas this is selecting don't exist when it runs. Might be better to add a `mentions` option to `TextEditor` like we were doing in INatTextArea and do this when *that* component mounts.